components/livesplit/liveSplitManager.ts

```diff
@@ -426,10 +431,21 @@ export class LiveSplitManager {
         this._timeCalcEntries.push(entry)
     }
 
-    private _generateTimeCalcLink() {
-        const url = new URL(
-            "https://solderq35.github.io/fg-time-calc/?mode=0&fs3=1&ft2=1&f3t1=1&f4t0=1&d=:&o1=1&fps=",
+    private _unsplitLastTimeCalcEntry() {
+        const entry = this._timeCalcEntries.pop()
+        entry.isCompleted = false
+        log(
+            LogLevel.DEBUG,
+            `Unsplitted TimeCalc entry: ${JSON.stringify(entry)}`,
         )
```

Comment:
probably get rid of this log before merging

Reply:
Removed!